docs/quickstart.md

```diff
@@ -110,7 +110,15 @@ def test_rouge2_similarity(generated_outputs, reference_outputs):
     assert langcheck.metrics.rouge2(generated_outputs, reference_outputs) > 0.9
 ```
 
-Coming soon: LangCheck can also help you create new test cases with `langcheck.augment`!
+LangCheck can also help you automatically create new test cases with `langcheck.augment`!
```

Comment:
nit: I feel like augment makes more sense to be mentioned in the `Using LangCheck` section rather than the `Use Cases > Unit Testing` section? Since it can be used for things other than unit testing.

Reply:
Moved!